Modules/Scripted/ShapeAnalysisModule/ShapeAnalysisModule.py

```diff
@@ -1372,18 +1371,19 @@ def setup(self):
       cli_parameters = {}
 
       #     IF Mesh To Label Map has been skipped AND the input given was already a label map
-      if self.skip_meshToLabelMap and not os.path.exists(LabelMapOutputFilepath):
-        inputFilepath = inputDirectory + '/' + self.CaseInput
-        labelmap_input_node = MRMLUtility.loadMRMLNode(inputFilepath, 'LabelMapVolumeFile')
+      if self.skip_meshToLabelMap and not os.path.exists(MeshToLabelMapOutputFilepath):
+        PossProcessInputDirectory = inputDirectory
+
+        labelmap_input_node = MRMLUtility.loadMRMLNode(self.inputRootname, inputDirectory, self.CaseInput, 'LabelMap')
       #     ELSE the input given was a model which has been transformed by MeshToLabelMap and store in the folder LabelMap
       else:
         labelmap_input_node = meshtolabelmap_output_node
-        inputFilepath = LabelMapOutputFilepath
+        PossProcessInputDirectory = MeshToLabelMapOutputDirectory
 
 
       cli_parameters["fileName"] = labelmap_input_node
 
-      pp_output_node = MRMLUtility.addnewMRMLNode("output_PostProcess", slicer.vtkMRMLLabelMapVolumeNode())
+      pp_output_node = MRMLUtility.createNewMRMLNode(PostProcessOutputRootname, slicer.vtkMRMLLabelMapVolumeNode())
```

Comment:
Second parameter should be a string.. Not that it matters, but that's what the function is expecting.